02-KernelModules_CharacterDevice/Makefile

```diff
@@ -0,0 +1,7 @@
+obj-m += chrdev.o
+
+all:
+	make -C /home/alex/Downloads/linux-5.0.7 M=$(PWD) modules
+
+clean:
+	make -C /lib/modules/$(shell uname -r)/build M=$(PWD) clean
```

Comment:
Need fix PATH

 There are no hard coded paths! You make out the variable, set the default value, give the opportunity to change the variable